schema/reference.yaml

```diff
@@ -2,8 +2,8 @@ title: Reference
 description: Reference defines a git clone reference.
 type: object
 properties:
-  name:
-    description: Name provides ref name.
+  value:
```

Comment:
I think we should keep this `name`, instead of value since this aligns with the official git terminology